content-webapp/src/main/webapp/vue-app/news/components/NewsApp.vue

```diff
@@ -107,8 +108,19 @@
           {{ $t('news.app.loadMore') }}
         </v-btn>
       </div>
+      <exo-confirm-dialog
+        ref="deleteConfirmDialog"
+        :message="confirmDeleteNewsDialogMessage"
+        :title="confirmDeleteNewsDialogTitle"
+        :ok-label="$t('news.button.ok')"
+        :cancel-label="$t('news.button.cancel')"
+        @ok="deleteByConfirm" />
       <news-activity-sharing-spaces-drawer />
       <activity-share-drawer />
+      <news-mobile-action-menu
```

Comment:
- this is only for mobile case no ?
- we should discuss about moving this from [news details](https://github.com/Meeds-io/content/pull/244/files#diff-323cd28dd45735acbe9b0044569d6edc1ed8df0e849f0855e773744756dd6e14L55), how it will be displayed for news detail case

Reply:
It is here : https://github.com/Meeds-io/content/pull/244/files#diff-25b72127791250ee9bc6fbf8fee6ef31d9b983e40300de883d944e76e0a69273R79

Reply:
and yes this for mobile case in the news App